app/models/game.rb

```diff
@@ -4,6 +4,16 @@ class Game < ActiveRecord::Base
 
 	after_create :populate_board!
 
+
+	def index
+		@list_all_games=Game.all
+	end
```

Comment:
Thanks Aaron, I'll get to changing the name today!

On Fri, Oct 2, 2015 at 10:24 AM, Aaron Washburn <notifications@github.com>
wrote:

> In app/models/game.rb
> <https://github.com/firehoseCheckmates/ourGame/pull/21#discussion_r41026573>
> :
>
> > @@ -4,6 +4,16 @@ class Game < ActiveRecord::Base
> >
> >  	after_create :populate_board!
> >
> > +
> > +	def index
> > +		@list_all_games=Game.all
> > +	end
>
> we typically just call this @games <https://github.com/games>. At a later
> point we may want to list only certain kinds of games and we can deal with
> naming then.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/firehoseCheckmates/ourGame/pull/21/files#r41026573>.
>